### ISSUE TITLE
Cloning Boxes

### ISSUE THREAD

**Issue by SaschaHagemann** [palasthotel/grid#315]
What about cloning boxes? 

Some boxes are quite complex and elaborate. Our customers Böll & MAN both asked, if there will be a possibility to clone boxes - in a sense of: using one box as a boilerplate / pattern / model / sample for building a similar box.

**Comment by EdwardBock** [comment]
@mkernel 
we should discuss this

**Comment by EdwardBock** [comment]
We will do this with #375 and give users the possibility to copy and past boxes, containers or even whole grids #323

**Comment by SaschaHagemann** [comment]
<3

**Comment by EdwardBock** [comment]
@mkernel 
Copy past functions are already running on node js async server right? Is there a ticket with specs?

**Comment by mkernel** [comment]
@EdwardBock yes, there is: https://github.com/palasthotel/Grid-Async/wiki/Grid-Async#pasteboard

it's simple: when you want to place sth. in the pasteboard simply use pasteboard.copy and provide the object. The content get's automatically pushed to all browser tabs with pasteboard.content.

Means: the content of the pasteboard should always be present in the js environment.

**Comment by EdwardBock** [comment]
Perfect!

**Comment by benthebear** [comment]
+1 for this feature from another project
